src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/ContributeModal.tsx

```diff
@@ -47,6 +48,8 @@ const ContributeModal: React.FC<Props> = ({
   const { t } = useTranslation()
   const valueWithTokenDecimals = new BigNumber(value).times(DEFAULT_TOKEN_DECIMAL)
 
+  const gasPrice = BIG_TEN.times(new BigNumber(10).pow(new BigNumber(9))).toNumber()
```

Comment:
You can use one more BIG_TEN  here :)

Reply:
Thanks, fixed in 44fbde2239d647eb7d59dd366400c4de7bb6b1da